models/user.go

```diff
@@ -31,3 +31,21 @@ func SaveUser(email string, password string) error {
 	}
 	return nil
 }
+
+func FindOneUserByCondition(condition interface{}) (User, error) {
```

Comment:
Minor naming improvement suggestion:

```suggestion
func FindUserBy(condition interface{}) (User, error) {
```

I find it descriptive and nicer to use 🌟

Reply:
Sounds better. Fixed in 752d3fca9f92953e4ecb829dc7104e2a353a218a